js-advanced-functions/fn-exercise-1.js

```diff
@@ -0,0 +1,65 @@
+//1. part one
+let initialData=[200,500,300,1000]
+console.log('INITIAL DATA: ',initialData)
+let newArray=[]
+console.log('Part 1 of the exercise -------------')
+
+const incrementing=(array,padding)=>{
+    for(i=0;i<array.length;i++){
+        array[i]=array[i]+padding
+        // console.log(array)
+    }
+    // for(el of array){
+    //     el=el+padding
+    //     // console.log(el)
+    // }
+    // the two for cycles do the same
+    newArray=array
+    console.log('Incrementing function by 2',newArray)
+}
+incrementing(initialData,padding=2)
```

Comment:
```suggestion
incrementing(initialData, 2)
```

Reply:
When calling a function the passing of the arguments is by passing a variable or an inline value. What was wrong here is the `padding = 2`